### ISSUE TITLE
CSS string for useCSS

### ISSUE THREAD

**Issue by dy** [streamich/react-use#184]
Hi! Awesome collection!
Is there any way to pass CSS string to useCSS?
Cheers!

**Comment by streamich** [comment]
Not now, but [`stylis`](https://github.com/thysultan/stylis.js) (the library usually used for CSS strings in JS) could be added to provide that functionality.

**Comment by dy** [comment]
Yeah, I saw the nano-css addon. Is there a simple way to set up the hopk with that addon?

**Comment by streamich** [comment]
Don't know how simple that it but it is basically 2 steps:

1. CSS strings needs to be run through `stylis`
2. Then `stylis` output needs to be converted to what [`nano.VSheet`](https://github.com/streamich/nano-css/blob/master/docs/vcssom.md) expects for the `.diff()` method.
